lib/commands/build_command.ts

```diff
@@ -17,7 +17,7 @@ export async function runBuildCommand(args: BuildCommand) {
   if (output.wasmFileName != null) {
     const wasmDest = path.join(args.outDir, output.wasmFileName);
     await Deno.writeFile(wasmDest, new Uint8Array(output.bindgen.wasmBytes));
-    if (args.isOpt) {
+    if (args.profile === "release" && args.isOpt) {
```

Comment:
Small point, but we should probably move this check to here and have it check `flags.debug`:

https://github.com/denoland/wasmbuild/blob/e1fd6ce68aedb15fcd99ff139e0307faa409ca66/lib/args.ts#L59

Reply:
Thank you for comments! I refactored in 31bb43d